# Type: docs
# Subtype: D2
# English Summary: Added version tag in Javadoc
# Chinese Summary: 在 javadoc 中添加了版本标签
# Entry Number: 2

# === Git Diff Content ===

diff --git a/subprojects/core/src/main/groovy/org/gradle/api/artifacts/ResolvableDependencies.java b/subprojects/core/src/main/groovy/org/gradle/api/artifacts/ResolvableDependencies.java
index 097cb95..d3c947d 100644
--- a/subprojects/core/src/main/groovy/org/gradle/api/artifacts/ResolvableDependencies.java
+++ b/subprojects/core/src/main/groovy/org/gradle/api/artifacts/ResolvableDependencies.java
@@ -87,6 +87,7 @@ public interface ResolvableDependencies {
      * that gives access to the graph of the resolved dependencies.
      *
      * @return the resolution result
+     * @since 1.3
      */
     @Incubating
     ResolutionResult getResolutionResult();
